### PR TITLE
[infra] Build docker image with ubuntu codename

### PR DESCRIPTION
**Pull request by hseok-oh**
Use ubuntu codename for docker build parameter
build-docker-image "--extension" parameter will be deprecated
Change dockerfile path by using ubuntu codename

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: infra/command/build-docker-image [hunk 1]
```diff
@@ -26,10 +29,20 @@ do
...
+      else
+        echo "Invalid extension"
+        exit 1
+      fi
       shift
       shift
       ;;
+    --codename)
+      UBUNTU_CODENAME=$2
+      DOCKER_TAG=$2
```
Comment (lemmaa) [inline infra/command/build-docker-image:43]: I cannot understand this line. Are the rules for generating image names different when using `--extension` and `--codename`?

Comment (hseok-oh) [reply infra/command/build-docker-image:43]: `--extension` didn't set image names. `--codename` will set tag automatically if user don't set name explicitly by using `--tag` parameter or `DOCKER_IMAGE_NAME` environment variable

Comment (lemmaa) [reply infra/command/build-docker-image:43]: I mean that the :`tag` part of image name, if I run this script with `--codename bionic`, the DOCKER_TAG will set as `bionic` here, so default image name will be `nnfw/nnas:bionic`, right?

Comment (hseok-oh) [reply infra/command/build-docker-image:43]: >so image name will be nnfw/nnas:bionic, right?

Right. 

I suggested to rename docker image (#4735), and it prepares renaming with ubuntu codename. When we support different architecture (ex. aarch64), tag name will include target architecture later.

Comment (lemmaa) [reply infra/command/build-docker-image:43]: Aha, I understood. I have missed that issue. 👍

**Review by lemmaa — APPROVED** [review]
LGTM!

**Pull request merged**